### DISCUSSION TITLE
Windows 11 ARM64 on UTM SE

### DISCUSSION THREAD

**Discussion by jridder** [utmapp/UTM#6492] [Windows]
I've been trying to install Windows 11 ARM64 on my M4 iPad Pro and I can boot but the lettering on Windows is all garbled/scrambled. Any idea how to resolve it?
![IMG_0005](https://github.com/user-attachments/assets/fff77862-fa74-49c0-b7e0-e2759254b2da)

**Comment by AppStudioLB** [comment]
Me too, I’m struggling with the same problem.

**Comment by a2she** [comment]
Me too…

**Comment by augustomaiorino** [comment]
Same problem please help!

**Reply by LHC0521** [reply]
Same，maybe use x86_64 can solve

**Comment by pedestriantactics** [comment]
Same. iPad Pro M4. Obtained the installer from CrystalFetch on a Mac.

**Comment by lgold01** [comment]
Did anyone get past this.  I am getting the same screen.
